externs/shaka/player.js

```diff
@@ -768,6 +769,13 @@ shaka.extern.DashManifestConfiguration;
  *   <code>EXT-X-PROGRAM-DATE-TIME</code> tags in the manifest.
  *   Meant for tags that are incorrect or malformed.
  *   <i>Defaults to <code>false</code>.</i>
+ * @property {string} mediaPlaylistFullMimeType
+ *   A string containing a full mime type, including both the basic mime type
+ *   and also the codecs. Used when the HLS parser parses a media playlist
```

Comment:
I think we should add a note about why this is needed.  For example, mention that HLS media playlists don't have this critical information.

Reply:
Done.